en/guide/overview.md

```diff
@@ -44,16 +44,18 @@ Advanced configuration information about individual cameras is specified in [Cam
 
 CSD implements the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html) for image and video capture and storage, and for getting/setting camera parameters and options.
 
-The MAVLink properties of CSD are specified in the *CSD Configuration File*:
-* The [\[mavlink\]](../guide/configuration_file.md#mavlink) section is used to specify the MAVLink destination UDP port, the broadcast address for heartbeat messages, and the system id (which should be set to match the autopilot).
+The MAVLink properties of CSD are specified in the [CSD Configuration File](../guide/configuration_file.md):
+* The [\[mavlink\]](../guide/configuration_file.md#mavlink) section is used to specify the MAVLink destination UDP port, the broadcast address for heartbeat messages, and (optionally) the system id.
 * The [\[uri\]](../guide/configuration_file.md#uri) section specifies the device to URI mapping for [Camera Definition File](../guide/camera_definition_file.md).
 * The [\[imgcap\]](../guide/configuration_file.md#imgcap) and [\[vidcap\]](../guide/configuration_file.md#vidcap) sections specify the *default settings* for image and video capture, respectively.
 
-Component IDs for each camera are allocated automatically and sequentially from [MAV_COMP_ID_CAMERA2](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA2) to [MAV_COMP_ID_CAMERA6](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA6) (inclusive) as cameras are connected (once all component ids are allocated further cameras are not addressable).
+All cameras share the same MAVLink system ID. This can (optionally) be defined in the configuration file to match the id of the associated autopilot. If the system id is not defined in the configuration file then CSD will initially use the default PX4 vehicle system ID (1) and then change to the id of the first autopilot it detects.
+
+Component IDs for each camera are allocated automatically and sequentially from [MAV_COMP_ID_CAMERA](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA) as cameras are connected (once all component ids are allocated further cameras are not addressable). For Gazebo-enabled CSD builds the first ID is allocated to the Gazebo plugin camera.

```

Comment:
> For Gazebo-enabled CSD builds the first ID is allocated to the Gazebo plugin camera
Not sure, what it means

Reply:
Thanks. I re-wrote as "When the CSD is built with Gazebo enabled, the Gazebo camera has component ID `MAV_COMP_ID_CAMERA`."